guide/english/voice/index.md

```diff
@@ -5,7 +5,7 @@ title: Voice
 # Voice
 Speech recognition allows users affected by accessibility difficulties (such as permanent visual impairment or temporary impairment while driving) the ability to navigate content on a website or input text data (such as a form). 
 
-Speech synthesis provides websites the ability to provide information to users by reading text.
```

Comment:
I prefer this sentence as it is less wordy and still gets the same message across

Reply:
I agree with @ahtee

Reply:
thanks, @ahtee for suggestions